Lombiq.Tests.UI/SecurityScanning/SecurityScanningUITestContextExtensions.cs

```diff
@@ -134,18 +205,27 @@ public static class SecurityScanningUITestContextExtensions
     /// <see href="https://www.zaproxy.org/docs/automate/automation-framework/"/> for details.
     /// </param>
     /// <param name="configure">A delegate to configure the security scan in detail.</param>
+    /// <param name="startUri">
+    /// The address where the scan should start. If <see langword="null"/> is used then the current browser location
+    /// will be used via <see cref="NavigationUITestContextExtensions.GetCurrentUri"/>.
+    /// </param>
     /// <returns>
     /// A <see cref="SecurityScanResult"/> instance containing the SARIF (<see
     /// href="https://sarifweb.azurewebsites.net/"/>) report of the scan.
     /// </returns>
     public static Task<SecurityScanResult> RunSecurityScanAsync(
         this UITestContext context,
         string automationFrameworkYamlPath,
-        Action<SecurityScanConfiguration> configure = null)
+        Action<SecurityScanConfiguration> configure = null,
+        Uri startUri = null)
```

Comment:
This parameter is used to start at a different URL [within the same run](https://github.com/Lombiq/UI-Testing-Toolbox/blob/a9a408350ad2b71733b9975ac13fdb92c4140f26/Lombiq.Tests.UI/SecurityScanning/SecurityScanningUITestContextExtensions.cs#L183-L197), specifically `~/Lombiq.Tests.UI/Error/Index`. That's regardless of the configuration.

Reply:
You can also achieve that by copying the config object and changing its start URL. I don't think it's useful to have this parameter in the public API for that internal use case.

Reply:
Sorry, I don't understand. Check for what being null? If you mean the start URL, we set it to the current URL by default, then it can be overridden in the `configure` action below so it's never null.

Reply:
Ah indeed. I thought you can pass in a `SecurityScanConfiguration` object.